docs/data/routes/docs/nextjs/internationalization/using-i18n/en.md

```diff
@@ -3,4 +3,78 @@ name: using-i18n
 routeTemplate: ./data/component-templates/article.yml
 title: Using Next.js i18n
 ---
-# Using Next.js i18n with JSS
\ No newline at end of file
+# Using Next.js i18n with JSS
+
+Next.js provides a way to use [Internationalized Routing](https://nextjs.org/docs/advanced-features/i18n-routing).
+The sample app is using [Sub-path Routing](https://nextjs.org/docs/advanced-features/i18n-routing#sub-path-routing).
+
+>  Domain routing requires custom implementation. 
+
+Let's dive into details on how the sample app enables `i18n`.
+
+> `jss export` doesn't support multilingual apps, in this case you should disable localization. Refer to [page](/docs/nextjs/deploying-to-production/export).
+
+### Initialization
+
+The file `next.config.js` contains definitions for `locales` and `defaultLocale`:
+* `locales` are all the locales you want to support in your application and should generally match (at least be a subset of) those in Sitecore.
+* `defaultLocale` is the locale used when visiting a non-locale prefixed path, such as `/styleguide`. It's `config.language` from your `package.json` file.
+
+```js
+i18n: {
+	locales: ['en', 'da-DK'],
+	defaultLocale: packageConfig.language,
+}
+```
+
+Next.js does not provide anything for localization, only i18n routing. To enable localization, the sample app uses the [next-localization](https://github.com/StarpTech/next-localization) library.
+
+> NOTE: If your app is not multilingual, you can remove `next-localization` and references to it.
+
+In `src/pages/_app.tsx`, the `App` initializes using `I18nProvider` with properties populated in `src/lib/page-props-factory.ts`:
+* `pageProps.locale` - provided by nextjs.
+* `dictionary` - fetched from dictionary service.
+
+The `DictionaryService` fetches translations in `src/lib/page-props-factory.ts`.
```

Comment:
Is this line necessary or can it be part of the bullet above? i.e. "* `dictionary` - fetched from the `DictionaryService`"